src/solvers/prop/cover_goals.cpp

```diff
@@ -48,13 +48,10 @@ void cover_goalst::constraint()
 
   // cover at least one unknown goal
 
-  for(std::list<goalt>::const_iterator
-      g_it=goals.begin();
-      g_it!=goals.end();
-      g_it++)
-    if(g_it->status==goalt::statust::UNKNOWN &&
-       !g_it->condition.is_false())
-      disjuncts.push_back(literal_exprt(g_it->condition));
+  for(const auto &g : goals)
+    if(g.status==goalt::statust::UNKNOWN &&
+       !g.condition.is_false())
+      disjuncts.push_back(literal_exprt(g.condition));
```

Comment:
I believe the next commit reformats this, could this commit already use the correct formatting?

Reply:
done